workshop/exercise-2/README.md

```diff
@@ -63,4 +63,119 @@ Before your continue, make sure all the pods are deployed and **`Running`**. If
 
 Congratulations! You successfully installed Istio into your cluster.
 
+
+## Validating Installation
+
+In this exercise, you will validate your Istio installation and launch some debugging containers to experiment with the service mesh.
+
+
+## Service Mesh Sidecar
+
+![istio architecture](istio-whatisit.svg)
+
+Istio's core functionality is achieved by setting up a proxy, or sidecar, inside every kubernetes pod. Outgoing and incoming traffic bound for the pod is intercepted by iptables and delivered to the `envoy` daemon running in the sidecar. The `envoy` proxy can then inspect, modify, forward, and record all traffic used by the application. East-West traffic goes through two `envoy` proxies, one at source and one at destination. All `envoy` proxies are configured by `pilot`, an Istio component.
+
+## Sidecar injection
+
+In Kubernetes, a sidecar is a utility container in the pod, and its purpose is to support the main container. For Istio to work, Envoy proxies must be deployed as sidecars to each pod of the deployment. There are two ways of injecting the Istio sidecar into a pod: manually using istioctl CLI tool or automatically using the Istio Initializer. In a previous step, we set up our cluster to use automatic injection.
+
+## Terminals
+
+In the upcoming exercises, it is useful to have multiple terminals open. Each will need the `KUBECONFIG` environment variable you exported earlier in the tutorial. If you get errors like:
+
+```bash
+kubectl get pod
+The connection to the server localhost:8080 was refused - did you specify the right host or port?
+```
+
+Go back and export that environment variable and try again.
+
+
+### Launch a debug container
+
+```bash
+kubectl run -i --tty poke2-`date +%s` --image=nibalizer/utilities --restart=ver -- sh
```

Comment:
if this image is useful we should consider moving it into the "ibmcom" namespace on dockerhub so we're not counting on a personal namespace